packages/terra-embedded-content-consumer/tests/nightwatch/providers/MessagingProvider.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Provider } from 'xfc';
+import './Provider.scss';
+
+class MessagingProvider extends React.Component {
+  constructor(props) {
+    super(props);
+    Provider.init({
+      acls: ['*'],
+      secret: 'OAuth Secret',
+      onReady: () => {
+        Provider.on('SHOW_STATUSES', () => {
+          const lifeCycleStatuses = ['Mounted', 'Launched', 'Authorized'];
```

Comment:
can we simplify this example... really all you need to do is acknowledge the event was received